addons/survey/views/survey_result.xml

```diff
@@ -96,21 +97,17 @@
                 <t t-set="text_result" t-value="prepare_result"/>
                 <tr t-foreach="text_result" t-as="user_input">
                     <td><a t-att-href="'%s/%s' % (user_input.user_input_id.print_url, user_input.user_input_id.token)"><t t-esc="user_input_index + 1"></t></a></td>
-                    <t t-if="question.type == 'free_text'">
-                        <td>
-                            <span t-field="user_input.value_free_text"></span><br/>
-                        </td>
-                    </t>
-                    <t t-if="question.type == 'textbox'">
-                        <td>
-                            <span t-field="user_input.value_text"></span><br/>
-                        </td>
-                    </t>
-                    <t t-if="question.type == 'datetime'">
-                        <td>
-                            <span  class="oe_date" t-field="user_input.value_date"></span><br/>
-                        </td>
-                    </t>
+                    <td>
+                        <t t-if="question.type == 'free_text'">
+                            <a t-att-href="'%s/%s' % (user_input.user_input_id.print_url, user_input.user_input_id.token)" t-field="user_input.value_free_text"></a><br/>
+                        </t>
+                        <t t-if="question.type == 'textbox'">
+                            <a t-att-href="'%s/%s' % (user_input.user_input_id.print_url, user_input.user_input_id.token)" t-field="user_input.value_text"></a><br/>
+                        </t>
+                        <t t-if="question.type == 'datetime'">
+                            <a t-att-href="'%s/%s' % (user_input.user_input_id.print_url, user_input.user_input_id.token)" class="oe_date" t-field="user_input.value_date"></a><br/>
+                        </t>
+                    </td>
```

Comment:
Is this pure refactoring, or did I miss something? If it is, please revert to avoid adding noise to code history